pom.xml

```diff
@@ -40,12 +40,12 @@
             <dependency>
                 <groupId>cz.kb.openbanking.adaa.client</groupId>
                 <artifactId>api</artifactId>
-                <version>1.0.0</version>
+                <version>1.1.0-SNAPSHOT</version>
             </dependency>
             <dependency>
                 <groupId>cz.kb.openbanking.adaa.client</groupId>
                 <artifactId>jersey-impl</artifactId>
-                <version>1.0.0</version>
+                <version>1.1.0-SNAPSHOT</version>
```

Comment:
Let's make this `1.1.0` version. It's not a `SNAPSHOT` anymore. Is it?

Reply:
After merge this PR, I'll release it to 1.1.0